cli/formatter/types.go

```diff
@@ -0,0 +1,120 @@
+package formatter
+
+// renderNode represents content for rendering.
+type renderNode interface{}
+
+const (
+	scalarRenderNodeType = iota
+	arrayRenderNodeType
+	mapRenderNodeType
+)
+
+// Opts contains formatting options.
+type Opts struct {
+	TransposeMode bool
+	NoGraphics    bool
+	ColWidthMax   int
+	TableDialect  TableDialect
+}
```

Comment:
It would be easier to view the package if each type will be in a separate file + a separate file for tests.
To `opts.go`.

Reply:
Done